http-server-netty/src/main/java/io/micronaut/http/server/netty/NettyRequestLifecycle.java

```diff
@@ -59,8 +59,6 @@ final class NettyRequestLifecycle extends RequestLifecycle {
         super(rib.routeExecutor);
         this.rib = rib;
         this.outboundAccess = outboundAccess;
-
-        multipartEnabled(rib.multipartEnabled);
```

Comment:
what is this replaced by

Reply:
Configuration is extracted from the route executor in `RequestLifecycle`.

Reply:
The check is coming from the shared config so I expect it's valid in the servlet